### ISSUE TITLE
Documentation: Use codelldb by Mason

### ISSUE THREAD

**Issue by theCollectiv** [mrcjkb/rustaceanvim#79]
### Feature description

There is a neat* way to set the codelldb path to Mason. Maybe this can improve the documentation:
[mrcjkb/rustaceanvim: Supercharge your Rust experience in Neovim! A heavily modified fork of rust-tools.nvim](https://github.com/mrcjkb/rustaceanvim#using-codelldb-for-debugging).

```lua
-- codelldb: config with Mason
local path = vim.fn.glob(vim.fn.stdpath("data") .. "/mason/packages/codelldb/extension/") or ""
local codelldb_path = path .. "adapter/codelldb"
local liblldb_path = path .. "lldb/lib/liblldb.so"
```
The advantage is, that codelldb updates do not have to be kept manually in sync or downloaded, if they are handled by Mason.

* originally found here:
[how to use rust-tools with mason.nvim? · Issue #2894 · LunarVim/LunarVim](https://github.com/LunarVim/LunarVim/issues/2894#issuecomment-1236420149)

**Comment by mrcjkb** [comment]
Thanks! I've updated the existing example in the readme :smile:

**Comment by mrcjkb** [comment]
Actually, come to think of it, mason is so common. I might as well automate that in the default config.

**Comment by mrcjkb** [comment]
Thanks for the update. Will be fixed in #82
